src/azul/service/avro_pfb.py

```diff
@@ -568,16 +568,24 @@ def _entity_schema_recursive(field_types: FieldTypes,
             }
         elif field_type in _nullable_to_pfb_types:
             # Exceptions are fields that do not become lists during aggregation
-            exceptions = (
+            field_exceptions = (
```

Comment:
Is this code only part of the old PFB manifest, or the verbatim one as well?

Reply:
Only the old manifest.

Reply:
Agreed at PL that this is fine.